### PR TITLE
fix(middleware): use only on text/html

### PR DESCRIPTION
**Pull request by cirotix**

### CODE REVIEW

File: tests/test_middleware.py [hunk 1]
```diff
@@ -14,16 +34,30 @@ def test_init(self):
...
+    def test_add_query_strings_to_links_is_not_called_for_pdf(self, mocker, request):
+        # GIVEN
+        mock_add_query_strings_to_links = mocker.patch(
+            "global_query_strings.middleware.add_query_strings_to_links"
+        )
+        middleware = GlobalQueryStringsMiddleware(dummy_pdf_middleware)
+        assert mock_add_query_strings_to_links.call_count == 0
+
+        # WHEN
+        response = middleware(request)
```
Comment (ThomSawyer) [inline tests/test_middleware.py]: You don't need to set the result in the `response` variable here. Same for  `test_add_query_strings_to_links_is_called_for_html`

File: tests/test_middleware.py [hunk 2]
```diff
@@ -1,11 +1,31 @@
 import pytest
+from django.http import HttpResponse
+from django.test import RequestFactory
```
Comment (ThomSawyer) [inline tests/test_middleware.py]: Here you don't need to import `RequestFactory`.

You can remove from `conftest.py` the following imports too:
```python
from django.contrib.auth.models import AnonymousUser
from django.contrib.sessions.middleware import SessionMiddleware
from django.test.client import RequestFactory
```

File: global_query_strings/middleware.py [hunk 3]
```diff
@@ -13,9 +13,8 @@ def __call__(self, request):
         # Handle image in Content-Type response
         # and handle StreamingHttpResponse response such as large CSV or SVG files.
```
Comment (ThomSawyer) [inline global_query_strings/middleware.py]: Maybe you can remove this comment now ?

**Review by ThomSawyer — CHANGES_REQUESTED** [review]
Some imports and a comment that can be removed.

To release the package you will need to:
* update the version in `setup.py`
* run `make release_prod`

File: tests/test_middleware.py (hunk 1 above)
Comment (cirotix) [reply tests/test_middleware.py]: fixed in 96de5d5

File: tests/test_middleware.py (hunk 2 above)
Comment (cirotix) [reply tests/test_middleware.py]: fixed in 7613e57

File: global_query_strings/middleware.py (hunk 3 above)
Comment (cirotix) [reply global_query_strings/middleware.py]: fixed in 476e193

**Comment by cirotix** [comment]
> To release the package you will need to:
> 
>     * update the version in `setup.py`

Done in 401adc2

**Review by ThomSawyer — APPROVED** [review]

**Pull request merged**